tests/dragonfly/replication_test.py

```diff
@@ -2033,13 +2033,19 @@ async def test_client_pause_with_replica(df_factory, df_seeder_factory):
     assert await seeder.compare(capture, port=replica.port)
 
 
+@pytest.mark.debug_only
+@dfly_args({"proactor_threads": 2})
 async def test_replicaof_reject_on_load(df_factory, df_seeder_factory):
     master = df_factory.create()
     replica = df_factory.create(dbfilename=f"dump_{tmp_file_name()}")
     df_factory.start_all([master, replica])
 
     c_replica = replica.client()
-    await c_replica.execute_command(f"DEBUG POPULATE 1000 key 1000 RAND type set elements 2000")
+
+    seeder = SeederV2(
+        key_target=1000, types=["SET"], data_size=150, collection_size=100, huge_value_size=0
```

Comment:
I am surprised that `SeederV2` is faster than `DEBUG POPULATE`. Is it the case?

Reply:
Not for all data types but the same configuration as we have here (and on debug) gives me:

1 passed, 116 deselected in 2.65s  -> `SeederV2`
1 passed, 116 deselected in 27.21s  -> `Debug populate`

Same config both seeders. running time mentioned is the seeding phase isolated and everything else removed (and a `dbfilename="" such that we don't snapshot at the end)

Reply:
Haven't looked on this any further but there are a few things I noticed:

1. sometimes seederv2 is faster than debug populate and vice versa (so it's not universal)
2. seeding with seederv2 makes it hard to reach the desired data size because of deletions and mofications of the key space. I tried to use seederv2 for example for rss test, and the same seeder config produced vastly different dataset sizes.

:shrug: